WcaOnRails/config/environments/test.rb

```
@@ -40,5 +40,5 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
```

I'd raise an exception in the test environment as well.